client/src/pages/BusinessList/BusinessList.css

```diff
@@ -0,0 +1,54 @@
+.BusinessList-Wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  margin-top: -10vh; /* Adds overlap styles to navbar */
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-position: bottom center;
+  background-image: url(../../assets//images/whitecrop\ 1.png);
```

Comment:
please use the hairwhite.jpg image, and follow the .home container (on home page) as far as your businessList-Wrapper specs. 
You need to incorporate the css grid and as per the figma docs (see attached screenshot).  Can do a zoom if you want.
no rounded corners on boxes, no shadows. 
![Screen Shot 2022-03-24 at 3 40 10 PM](https://user-images.githubusercontent.com/15850286/159997109-140e0b95-fc65-459e-b65e-8637fba2d642.jpg)

Reply:
Hi @cynthiaeddy! I do not see a `hairwhite.jpg image` in the figma files. The image named in the provided picture and the one I used is `whitecrop 1`. If there is a different one that should be used please add it to the figma and I will update it. Thank you!